core/shared/src/main/scala-2/org/log4s/package.scala

```diff
@@ -1,9 +1,9 @@
 package org
 
-import language.experimental.macros
+import scala.language.experimental.macros
 
 package object log4s extends log4s.`package-platform` {
-  def getLogger: org.log4s.Logger = macro LoggerMacros.getLoggerImpl
+  def getLogger: Logger = macro LoggerMacros.getLoggerImpl
   def getLogger(name: String) = new Logger(org.slf4j.LoggerFactory.getLogger(name))
   def getLogger(clazz: Class[_]) = new Logger(org.slf4j.LoggerFactory.getLogger(clazz))
```

Comment:
I tried extracting a `package-compat` for `getLogger`, but the tests no longer recognized the nullary overload.  The three need to be defined together, which means the `String` and `Class` versions get copied for now.